### PR TITLE
Feature/RECAPP-875 i18n

### PR DESCRIPTION
**Pull request by elwinvaneede**
You can test it by editing `lang="en"` in the `index.html` (supported languages: `en`, `de`, `fr`, and `nl`). If the lang attribute is omitted it will fallback on the browser language.

### CODE REVIEW

**Review by ian-w2w — APPROVED** [review]
Hm, fancy. No translation package! Elegant solution ✔️  
Did you use the supplied translations? (I see some sentences that raise questions, but might just be what the customer wants..?)

File: src/translations.ts [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+interface Translations {
+  de: Translation;
+  en: Translation;
+  fr: Translation;
+  nl: Translation;
+}
+
+const defaultLanguage: 'en' = 'en';
+const suportedLanguages: string[] = ['de', 'en', 'fr', 'nl'];
```
Comment (Robert-Smagge) [inline src/translations.ts:13]: `supportedLanguages` met dubbel-p, toch? ;-)

File: src/translations.ts [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+const translations: Translations = {
+  de: {
+    noConnection: 'Sie konnten keine Verbindung zum RecycleClub-Rechner herstellen.',
+    searchByCartridgenumber: 'Suche nach Patronennummer',
+    searchByBrand: 'Suche nach Marke',
+    forExampleAbbreviation: 'Z.B.',
+    cartridigeNotListed: 'Können Sie Ihre Patrone nicht im Rechner finden?',
```
Comment (Robert-Smagge) [inline src/translations.ts:21]: Hier staat een `i` teveel in `cartridge`

File: src/translations.ts [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    cartridigeNotListedRemarks:
```
Comment (Robert-Smagge) [inline src/translations.ts:22]: idem

**Review by Robert-Smagge — APPROVED** [review]
Alleen een paar typo's

**Review by denniscoorn-w2w — APPROVED** [review]
Awesome, nicely done 👍

**Comment by Robert-Smagge** [comment]
Testing! 🔬

**Comment by elwinvaneede** [comment]
> Hm, fancy. No translation package! Elegant solution ✔️
> Did you use the supplied translations? (I see some sentences that raise questions, but might just be what the customer wants..?)

Yes I did, except for those where there were no translations supplied. But yeah, I found DeepL would generate better translations than the supplied ones.

**Pull request merged**